CLIrefactor.py

```diff
@@ -7,104 +7,115 @@
-                        print("User letters: " + self.controller.controllerGetLetters())
-                        print("Req letters: " + self.controller.controllerGetReqLetter())
-                        self.controller.controllerUpdatePuzzleState1()
-                    if (isAuto.lower() == "no"):
-                        self.controller.controllerRunBaseGame()
-                        self.controller.controllerUpdatePuzzleState1()
-                
-                case "!showpuzzle":
-                        #lots of these are just printing stuff can be removed, just for testing purpsoe.
-                        print("Your letters: " + self.controller.controllerGetLetters())
-                        print("Required letter: " + self.controller.controllerGetReqLetter())
-                        print("Guessed words: " + str(self.controller.controllerGetGuessedWords()))
-                        print("Word Bank: " + str(self.controller.controllerGetWordList()))
-                        print("User Points: " + str(self.controller.controllerGetPoints()))
-                        print("Max points possible: " + str(self.controller.controllerGetPuzzleTotal()))
-                case "!showfoundwords":
-                        print("Guessed words: " + str(self.controller.controllerGetGuessedWords()))
-                case "!shuffle":
-                        self.controller.controllerShuffleAuto()
-                        print("Your letters: " + self.controller.controllerGetLetters())
-                case "!savepuzzle":
-                        inputFile = input("Please enter a name for the file: ")
-                        self.controller.controllerSaveGame(inputFile)
-                case "!loadpuzzle":
-                        inputFile = input("Enter the name of the file you want to load: ")
-                        checkFile = inputFile + ".json"
-                        if (os.path.exists(checkFile)):
-                            self.controller.controllerGameLoad(inputFile)
-                            print("Puzzle loaded!")
-                        else:
-                            #Tell the user the file doesn't exist
-                            print("Uh-oh! Couldn't find that file. Reenter the load command and try again.")
-                case "!showstatus":
-                        print("Rank: " + self.controller.controllerGetPuzzleRank())
-                case "!help":
-                       if(self.controller.controllerGetPuzzleState() == 0):
-                        self.controller.controllerStartCommands()
-                       else:
-                        self.controller.controllerHelpCommand()
-                case "!exit":
-                        exit()
-          
-view = view()
-view.startGame()
\ No newline at end of file
+                            self.controller.controllerHelpCommand()
+                    case "!exit":
+                            exit()
+            
```

Comment:
basically around here if where you'd check for the CLI arguement outside of the view class, if theres -cli you can run view = view() and view.startGame(), else just as you have it

Reply:
I'm not sure if an error would occur here since I haven't run it, but the above might work.